### ISSUE TITLE
UI 4

### ISSUE THREAD

**Issue by kxm27** [bimalkaf/Flutter_SinglePages#2]
can i please get an update on how to add an eye icon in the password field which makes the password visible or invisible.

thank you.

**Comment by raheelashabir** [comment]
yes if you want to put the icon before text then use prefix **prefixIcon: Icon(Icons.lock)** and if you want to put the icon after text then use suffix  **suffixIcon: Icon(Icons.lock)** and for the eye icon prefixIcon: Icon(Icons.remove_red_eye)
